ssl/statem/statem_clnt.c

```diff
@@ -1722,10 +1723,14 @@ MSG_PROCESS_RETURN tls_process_server_hello(SSL *s, PACKET *pkt)
         memcpy(labelbuffer, DTLS1_SCTP_AUTH_LABEL,
                sizeof(DTLS1_SCTP_AUTH_LABEL));
 
+        labellen = strlen(labelbuffer);
```

Comment:
Would `sizeof(DTLS1_SCTP_AUTH_LABEL)  - 1` be better here? I'm thinking that (unless the compiler is clever enough to optimise it away), `strlen` is a runtime call, whereas `sizeof` can be resolved at compile time. Same comment for the various other instances of this below.